### PR TITLE
Use the new @centrifuge/tinlake-js package

### PR DESCRIPTION
**Pull request by offerijns**

### CODE REVIEW

File: containers/Overview/index.tsx [hunk 1]
```diff
@@ -20,10 +19,10 @@ import { withRouter, NextRouter } from 'next/router'
 interface Props {
   tinlake: any
   loans?: LoansState
-  loadLoans?: (tinlake: Tinlake) => Promise<void>
+  loadLoans?: (tinlake: typeof Tinlake) => Promise<void>
```
Comment (philipstanislaus) [inline containers/Overview/index.tsx:22]: 😍

File: ducks/loans.ts [hunk 2]
```diff
@@ -55,7 +54,7 @@ export default function reducer(state: LoansState = initialState, action: AnyAct
 }
 
 // hardcoded root just for testing - will be removed in next pr
-export function loadLoans(tinlake: Tinlake): ThunkAction<Promise<void>, LoansState, undefined, Action> {
+export function loadLoans(tinlake: any): ThunkAction<Promise<void>, LoansState, undefined, Action> {
```
Comment (philipstanislaus) [inline ducks/loans.ts:57]: What happened to typing here?

File: components/WithTinlake/index.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
 import { initTinlake } from '../../services/tinlake'
-import { ITinlake } from 'tinlake'
+import { ITinlake } from '@centrifuge/tinlake-js'
 
 interface Props {
   render: (tinlake: any) => React.ReactElement
```
Comment (philipstanislaus) [inline components/WithTinlake/index.tsx:6]: Can we type it properly here now?

File: containers/Overview/index.tsx [hunk 4]
```diff
@@ -20,10 +19,10 @@ import { withRouter, NextRouter } from 'next/router'
 interface Props {
   tinlake: any
```
Comment (philipstanislaus) [inline containers/Overview/index.tsx:20]: And here as well?

**Review by philipstanislaus — APPROVED** [review]
LGTM, just a few comments. Thanks!

File: ducks/loans.ts (hunk 2 above)
Comment (offerijns) [reply ducks/loans.ts:57]: `tinlake.contractAddresses` doesn't seem to be exported somehow (or at least i couldn't figure out how to access it)

**Review by philipstanislaus — APPROVED** [review]
👍

**Pull request merged**